stock_move_actual_date/wizard/stock_quantity_history.py

```diff
@@ -18,3 +19,10 @@ def open_at_actual_date(self):
         ]
         action["display_name"] = format_datetime(self.env, self.inventory_datetime)
         return action
+
+    def open_at_actual_date_location(self):
```

Comment:
No need to have `location` in the method name.
```suggestion
    def open_qty_at_actual_date(self):
```